pkg/database/token.go

```diff
@@ -312,16 +316,16 @@ func (db *Database) PurgeTokens(maxAge time.Duration) (int64, error) {
 
 // updateStatsCodeInvalid updates the statistics, increasing the number of codes
 // that were invalid.
-func (db *Database) updateStatsCodeInvalid(authApp *AuthorizedApp) {
-	now := timeutils.UTCMidnight(time.Now().UTC())
+func (db *Database) updateStatsCodeInvalid(t time.Time, authApp *AuthorizedApp) {
+	t = timeutils.UTCMidnight(t.UTC())
```

Comment:
you don't need to do this here since UTCMidnight does this.

https://github.com/google/exposure-notifications-server/blob/main/pkg/timeutils/midnight.go#L28

Reply:
Done